flink-core/src/test/java/org/apache/flink/api/common/operators/ResourceSpecTest.java

```diff
@@ -35,23 +35,7 @@
 public class ResourceSpecTest extends TestLogger {
 
 	@Test
-	public void testIsValid() throws Exception {
-		ResourceSpec rs = ResourceSpec.newBuilder(1.0, 100).build();
-		assertTrue(rs.isValid());
-
-		rs = ResourceSpec.newBuilder(1.0, 100).
-				setGPUResource(1).
-				build();
-		assertTrue(rs.isValid());
-
-		rs = ResourceSpec.newBuilder(1.0, 100).
-				setGPUResource(-1).
-				build();
-		assertFalse(rs.isValid());
-	}
-
-	@Test
-	public void testLessThanOrEqual() throws Exception {
+	public void testLessThanOrEqual_BothSpecified() {
```

Comment:
I thought it was valid because I saw this pattern in some other tests, e.g. [DefaultDispatcherRunnerTest](https://github.com/zhuzhurk/flink/blob/master/flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/runner/DefaultDispatcherRunnerTest.java).
I'm fine to change it and the name case is also not correct.

Reply:
done.